packages/@atjson/renderer-commonmark/test/__snapshots__/table.test.ts.snap

```diff
@@ -0,0 +1,61 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`tables jagged table 1`] = `
+"
+|          |        |       |                                                                                                                                                                               |
+| -------- | ------ | ----- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| laios    |        |       |                                                                                                                                                                               |
+| marcille | 500    | mage  |                                                                                                                                                                               |
+| falin    | healer |       |                                                                                                                                                                               |
+| chilchuk | 29     | thief | 
+        Looks like a child but is actually a divorced father of three. He is
+        serious about his work and isn't interested in getting close with
+        people
+       |
```

Comment:
Hmm what's the newlines doing here?

Reply:
This is technically wrong here— I'm just not sure whether this is a test issue or something that needs to be fixed for markdown rendering.

Reply:
ohh good catch, it's a bug with markdown rendering. I need to make sure it strips newlines from the text in the markdown renderer